functions/src/index.ts

```diff
@@ -117,17 +120,20 @@ app.get("/go/:id", (req, res) => {
             <meta property="twitter:title" content="${title}"/>
             <meta property="twitter:description" content="${description}"/>
             ${
-              image &&
-              `<meta property="og:image" content="${image}" />
+              image
+                ? `<meta property="og:image" content="${image}" />
```

Comment:
using `{image && ...}` was rendering the string `undefined` on the redirect page if these weren't provided